File: problemas/1000/1073.cs
using System;

namespace uri1073
{
    internal static class Program
    {
        private static void Main()
        {
            int n = int.Parse(Console.ReadLine());

            for (int i = 2; i <= n; i += 2)
            {
                Console.WriteLine($"{i}^2 = {Math.Pow(i, 2)}");
            }
        }
    }
}

File: problemas/1000/1073.dart
import 'dart:io';
import 'dart:math';

void main() {
  final n = int.parse(stdin.readLineSync());

  for (var i = 2; i <= n; i += 2) {
    stdout.writeln('$i^2 = ${pow(i, 2)}');
  }
}